staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -808,7 +808,7 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 		}
 		printLabelsMultiline(w, "Labels", pod.Labels)
 		printAnnotationsMultiline(w, "Annotations", pod.Annotations)
-		if pod.DeletionTimestamp != nil {
+		if pod.DeletionTimestamp != nil && pod.Status.Phase == corev1.PodRunning {
```

Comment:
Should we also include Pending here for consistency?

Reply:
Done.